explainer/fenced_frame_config.md

```diff
@@ -0,0 +1,35 @@
+# Fenced frame configs
+
+
+# Introduction
+
+For use cases involving APIs that access cross-site data, we need to be able to load a fenced frame with content determined by the API without revealing information about the content to the embedding context. For example, with interest-based ads in [FLEDGE](https://github.com/WICG/turtledove), the winning ad that's returned from the auction auction depends on the user's cross-site interest group data, which we don't want to expose to the site that calls the auction. This document proposes a web-platform way of loading content into a fenced frame using an opaque object.
+
+
+# Proposed solution
+
+`<fencedframe>` has an attribute `config`, rather than `src`. APIs like FLEDGE return a WebIDL object called `FencedFrameConfig`, which has a series of fields that specify the behavior desired by the API (e.g. the ad url, width and height as seen from within the fenced frame, etc.). When the embedder stores this object into the `config` attribute, the fenced frame loads the context accordingly.
```

Comment:
It's really just a JS object, so I'd probably go for "return a `FencedFrameConfig` object defined by [our WebIDL](https://wicg.github.io/fenced-frame/#fenced-frame-config-interface)". The spec is obviously not finished, but that's the link to use if we wanted to add one (I think it's a good idea).